ssl-config-generator/index.html

```diff
@@ -632,8 +629,8 @@
     <script>
         (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+        m=s.getElementsByTagName(o)[0];a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```

Comment:
I agree, but script-inserted scripts are asynchronous already. `scriptElement.async = true` [literally does nothing except in Firefox 3.6](https://mathiasbynens.be/notes/async-analytics-snippet#async).